default.nix

```diff
@@ -101,7 +103,20 @@ stdenv.mkDerivation rec {
 
   installCheckTarget = [ "check" ];
 
-  passthru = { inherit ocamlPackages; };
+  passthru = {
+    inherit coq-version ocamlPackages;
+    dontFilter = true; # Useful to use mkCoqPackages from <nixpkgs>
+  };
+
+  setupHook = writeText "setupHook.sh" ''
+    addCoqPath () {
+      if test -d "$1/lib/coq/${coq-version}/user-contrib"; then
+        export COQPATH="$COQPATH$\{COQPATH:+:}$1/lib/coq/${coq-version}/user-contrib/"
```

Comment:
The escaping looks wrong. Please double-check.

Reply:
Hum, I had fixed this already, so I must have messed up with the force-pushing at some point.